core/services/pg/advisory_lock_test.go

```diff
@@ -114,8 +118,6 @@ func Test_AdvisoryLock(t *testing.T) {
 		require.NoError(t, err)
 
 		assert.True(t, exists)
```

Comment:
We might get weird errors because I believe the TakeAndHold loop now can persist even after the test exits

Reply:
Agreed, I realized I should not change the locks strategy and return `release` function. Will fix this today.